srcpkgs/qtlanguageserver/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'qtlanguageserver'
+pkgname=qtlanguageserver
+version=6.5.0
+revision=1
+build_style=cmake
+hostmakedepends="qt6-base"
+makedepends="qt6-base-devel"
+depends="qt6-base"
```

Comment:
is this not auto-detected?

Reply:
I am not sure what you mean by auto-detected but qt6-base is needed as a host package for cross compilation. qt6-base-devel is needed for compilation against qt6-base. qt6-base is needed at minimum as a runtime dependency because qtlanguageserver is used with qt6.

Reply:
if a program dynamically links to a library, xbps-src will detect it as a dependency automatically so you don't need to specify it

Reply:
I did not know that! Cool feature, probably through ldd right? If it is not in the Manual.md(which I don't think it is) whoever is generally in charge of that should consider adding a note about it. 

I removed the depends line.

Reply:
Even if misplaced a bit, it's described in https://github.com/void-linux/void-packages/blob/master/Manual.md#deps_runtime and mentioned in two other places.